examples/parachain-example/src/main.rs

```diff
@@ -18,9 +18,11 @@ impl Config for StatemintConfig {
     type Signature = <PolkadotConfig as Config>::Signature;
     type Hasher = <PolkadotConfig as Config>::Hasher;
     type Header = <PolkadotConfig as Config>::Header;
-    type ExtrinsicParams = <SubstrateConfig as Config>::ExtrinsicParams;
+    type ExtrinsicParams = StatemintExtriniscParams<Self>;
 }
 
+pub type StatemintExtriniscParams<T> = DefaultExtrinsicParams<T>;
+pub type StatemintExtrinsicParamsBuilder<T> = DefaultExtrinsicParamsBuilder<T>;
```

Comment:
Not sure what you mean with "tip". 

The error I got:
![Screenshot 2023-10-30 at 13 56 42](https://github.com/paritytech/subxt/assets/93204684/1a91241a-831c-416c-99db-522d9672f400)

In addition, in the `README.md` it says you can run this example by running:
`cargo run --bin parachain-example`

But this doesn't work. My PR doesn't include the solution for this. Only for running the example successfully.

Reply:
Aah ok, I see, thank you!

Since the config is actually identical to `PolkadotConfig` now, let's simplify first lines of this file to:

```rust
use subxt::{
    PolkadotConfig,
    utils::{AccountId32, MultiAddress},
    OnlineClient,
};
use subxt_signer::sr25519::dev::{self};

#[subxt::subxt(runtime_metadata_path = "statemint_metadata.scale")]
pub mod statemint {}

// PolkadotConfig or SubstrateConfig will suffice for this example at the moment,
// but PolkadotConfig is a little more correct, having the right `Address` type.
type StatemintConfig = PolkadotConfig;

#[tokio::main]
pub async fn main() {
 // ....
}
```

Reply:
`cargo run` will work too then :). I thought it would be possible to run it from the root of the repo.

I will add your proposed changes!

Reply:
True; I guess at least the current command makes sure it won't work at all if ran from the wrong folder :D